src/main/scala/package.scala

```diff
@@ -167,7 +167,8 @@ package object constellation extends POWExt
     do {
       retries += 1
       done = t
-      Thread.sleep(delay)
+      val normalizedDelay = delay + Random.nextInt((delay * (scala.math.pow(2, retries))).toInt)
```

Comment:
Don't know enough about the context, but just pointing out that `scala.math.pow(2, retries)` probably doesn't need those brackets around it :)